docs/api/1.x/index.rst

```diff
@@ -3,6 +3,30 @@
 1.x
 ###
 
+Full reference
+==============
+
+Full detailed API documentation:
+
+.. toctree::
+   :maxdepth: 2
+
+   cliquet/utilities
+   authentication
+   cliquet/batch
+   cliquet/timestamps
+   cliquet/backoff
+   cliquet/errors
+   cliquet/deprecation
+   buckets
+   collections
+   records
+   groups
+   permissions
+   synchronisation
+   cliquet/resource
```

Comment:
Done in https://github.com/Kinto/kinto/pull/574/commits/7737ff700a6a8764aae52c2d71fd2df7c4328854.